assets/components/countdown/countdown.jsx

```diff
@@ -0,0 +1,89 @@
+// @flow
+
+// ----- Imports ----- //
+import React, { Component } from 'react';
+
+
+// ---- Types ----- //
+type CountdownTime = {
+  t: number,
```

Comment:
What would be `t`?

Reply:
the unix time remaining. Should I make the variable clearer?

Reply:
if you could give it a more meaningful name would be great 👍